Header header
string mode
gigatron/Drive drive
bool estop
